.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Test Charts
```

Comment:
Minimize name repetition to maximize the use of space in status check names. I'd say you can drop "charts" from the names (workflow and job), since that's implied by the repository name. `test/test` (same workflow/job name) is unavoidable and I wouldn't worry about that.